app/src/main/res/layout-sw600dp/activity_oscilloscope.xml

```diff
@@ -54,8 +57,10 @@
                 tools:layout_editor_absoluteY="0dp"
                 app:layout_constraintLeft_toRightOf="@+id/tv_graph_label_xaxis_os"
                 android:layout_marginStart="8dp"
+                android:layout_marginLeft="8dp"
```

Comment:
Hello @akarshan96 ! Shall we add dimensions to `dimens.xml` so that we can handle multiple screen sizes from dimension files? :)